pkg/kudoctl/util/repo/package.go

```diff
@@ -157,27 +195,14 @@ func untarV1Package(r io.Reader) (*v1Package, error) {
 				return nil, errors.Wrapf(err, "while reading file from bundle tarball %s", header.Name)
 			}
 
-			err = readPackageFile(header.Name, bytes, &result)
+			err = readV1PackageFile(header.Name, bytes, &result)
 			if err != nil {
 				return nil, err
 			}
 		}
 	}
 }
 
-func isFrameworkV1File(name string) bool {
-	return strings.HasSuffix(name, frameworkV1FileName)
-}
-
-func isTemplateV1File(name string) bool {
-	matched, _ := regexp.Match(templateV1FileNameRegex, []byte(name))
-	return matched
-}
-
-func isParametersV1File(name string) bool {
-	return strings.HasSuffix(name, paramsV1FileName)
-}
-
 type v1Package struct {
```

Comment:
I know this is not part of this PR and was an earlier but having this down here instead of on the top feels odd to me.